linodecli/baked/request.py

```diff
@@ -103,7 +110,7 @@ def __init__(
             )
 
 
-def _parse_request_model(schema, prefix=None, parent=None):
+def _parse_request_model(schema, prefix=None, parent=None, depth=0):
```

Comment:
We calculate the depth here because we don't want to calculate the depth at runtime